### ISSUE TITLE
The script 'app-configuration-script-sh' try extract a non existent file shallalist.tar.gz

### ISSUE THREAD

**Issue by jpocampor** [Librerouter/Librekernel#285]
When I run the **app-configuration-script-sh*, and this execute the process of configure undbound dns server, the script try decompress the file /opt/shallalist.tar.gz (in line 1971), but the file not exists int this place. I have encountered that script is commented in line 1688, function configure_blacklists, and here is the only one place where it try download this file.

**Personal fix sugested:**
I try replace the line 1971 of the script, where this execute the **cp** instruction that copy the tar file on the current location with the **wget** command to download the file in the works directory and this run fine. This solution  has proposed bassed in my premature general project knowledge.
I attach the screenshot of issue and sugested fix.

I like heard your opinion of this. Thanks.

![shallalist](https://cloud.githubusercontent.com/assets/6977526/24839302/53a9006c-1d1d-11e7-8d04-4dd9eef93962.png)
![shallalist_2](https://cloud.githubusercontent.com/assets/6977526/24839305/58dc2f32-1d1d-11e7-9a53-f68002dc6c4d.png)
![shallalist_3](https://cloud.githubusercontent.com/assets/6977526/24839310/5c0a4aea-1d1d-11e7-9ccc-368c7785fb0f.png)

**Comment by JFrancoJ** [comment]
app-installation-script.sh line 612
wget -P /opt/ http://www.shallalist.de/Downloads/shallalist.tar.gz

I don't see any error in your reports. Expected error in your case in libre_install.log is "Unable to download domain list"

**Comment by jpocampor** [comment]
Ups, then I think that error, and other errors that I have had, is because the setup.sh script don't stop when an error occur in app-installation-script.sh, and in this case, the process continue and fail continously in app-configuration-script.sh, because don't stop in previous script. More, I think that error in relation to space disk, ram and cores number in the VM, don't stop although this are writing in logs.
It, generate a new question for me: ¿This is a new issue to report here?

**Comment by JFrancoJ** [comment]
Yes, check last yestarday commit of setup.sh that will alert in event 
installation-script no success


Juan Ocampo wrote:

**Comment by JFrancoJ** [comment]
Hi Juan
I did a commit yesterday on setup.sh that will stop 
configuration-script.sh in the event installation-script.sh ended with 
critical error.


Juan Ocampo wrote:

**Comment by jpocampor** [comment]
Hi @JFrancoJ 

Thanks, I will try again to install the project today.
I believe that all my installation issues are derived of this, because, I have could read all errors of disk space and ram on log that you tell me in other issues, but neither error has stoppped the installation process.

**Comment by orwellencarnation** [comment]
What you need to fix it? you are the expert...supossely
How you plan to debug it?

**Comment by jpocampor** [comment]
Hi @orwellencarnation 
Excuse me?, may be that if you read all previous comments, you could see that all doubts have been cleared up, and the error were in relationship to the setup.sh script execution and the not notification and/or stop execution in error case of the other scripts.

**Comment by nikdavnik** [comment]
Please give me some time. I am going to do some major changes in scripts. I will let you know when you can run scripts again

**Comment by jpocampor** [comment]
Hi @nikdavnik 
Ok, only I tell you that today finally I can run the script completely, and only have encountered a bit issues:

1. In debian 8, the cpupower package isn't available as default package.
2. At the end of execution, the process first print: "Installaation completed" and in the next line print "Installation Fail, please check log /var/libre_install.log". But apparentely, the installation process is successfull. I do a test of this soon and write the results.

Attachment the snapshot  (Left side is te execution script and its outputs, right side is the /var/libre_install.log content).
![captura de pantalla de 2017-04-11 23-07-43](https://cloud.githubusercontent.com/assets/6977526/24941267/b3667b40-1f0e-11e7-8029-a85e59715986.png)

And the next is the screenshot of the unavailable default package for cpupower:

![captura de pantalla de 2017-04-11 20-16-44](https://cloud.githubusercontent.com/assets/6977526/24941283/db55bfd0-1f0e-11e7-8c13-02d298dffc79.png)

**Comment by JFrancoJ** [comment]
I am going to commit a change in app-installation-script.sh
There a misstyped "InstallAAtion" ( two a )



Juan Ocampo wrote:

**Comment by orwellencarnation** [comment]
David Franco which major shit your talking about inform with details and do not fuck use the word major!

**Comment by orwellencarnation** [comment]
Commited? fixed? other related issues?

**Comment by JFrancoJ** [comment]
fixed and commited , was related with some missed depencencies for dnscrypt

orwellencarnation wrote:

**Comment by orwellencarnation** [comment]
which commit number?

**Comment by orwellencarnation** [comment]
why you dont reference them when you fix issues? you want to do same crap as david?
